docs/archive/examples/zoom-activity-dashboard.md

```diff
@@ -246,8 +246,6 @@ The rest of the charts will be needing the **webinars** and `report_webinar_part
 
 For this chart, as for the meeting’s counterpart, we will get a calculated field off the Duration field to get the **Webinar Duration in Hours**, and then plot **Created At** against the **Sum of Webinar Duration in Hours**, as shown in the screenshot below. Note: Make sure you create a new sheet for each of these graphs.
 
-![](../../.gitbook/assets/duration-spent-in-weekly-webinars%20(3)%20(3).png)
```

Comment:
is this image important here? we can probably fix this if necessary.

btw HTML standard replaces spaces like " " with "%20" which is why it says `webinars%20(3)` instead of `webinars (3)` because spaces aren't allowed in URLs/URIs (technical names for web addresses)

Reply:
Nah I removed the image. The file is in the archive folder anyways.